### ISSUE TITLE
ES6 exponentials (**) cause error:

### ISSUE THREAD

**Issue by sblackstone** [jshint/jshint#3146]
Code: 

```
/* jshint esversion: 6, node: true */
'use strict';

let a = 5;

console.log(a**2);
```

Error:

```
Five warnings
6	Expected an identifier and instead saw '*'.
6	Expected ')' and instead saw '2'.
6	Missing semicolon.
6	Expected an identifier and instead saw ')'.
6	Expected an assignment or function call and instead saw an expression.
```

Expected it to pass as this is a valid ES6 construct..

Thanks!

**Comment by sblackstone** [comment]
Just found #2602 which seems to discuss this issue from before its standardization...

**Comment by jugglinmike** [comment]
The exponentiation operator was not standardized until version 7 of ECMAScript, which is why `esversion: 6` does not support it. Support for ES7, enabled via `esversion: 7`, [has been merged to the `v2.10.0` branch](https://github.com/jshint/jshint/pull/3060) and will be available in the next major release of JSHint. We do not currently have an expectation for the release date, so stay tuned.
